cirq/sim/clifford/act_on_clifford_tableau_args.py

```diff
@@ -86,8 +86,7 @@ def _act_on_fallback_(self, action: Any, allow_decompose: bool):
 
 def _strat_act_on_clifford_tableau_from_single_qubit_decompose(
         val: Any, args: 'cirq.ActOnCliffordTableauArgs') -> bool:
-    num_qubits_getter = getattr(val, 'num_qubits', None)
-    if num_qubits_getter is not None and num_qubits_getter() == 1:
+    if num_qubits(val) == 1:
```

Comment:
This change should be technically split out to a separate PR - do you mind doing that?

Reply:
Makes sense. I'll split out the small changes that don't really fit here and send separate PRs for them. Will hold off on this one for now since the small ones would need to go in first.